src/Components/Publishing/Byline/Byline.tsx

```diff
@@ -19,10 +19,10 @@ const Byline: React.SFC<BylineProps> = props => {
   const share = layout === "condensed"
     ? false
     : <Share
-        url={`http://www.artsy.net/article/${article.slug}`}
-        title={article.social_title || article.thumbnail_title}
-        color={layout === "fullscreen" ? "white" : "black"}
-      />
+      url={`http://www.artsy.net/article/${article.slug}`}
```

Comment:
Ah we should be using `Constants.articleHref()` now

Reply:
Sure I can add that